target/linux/ar71xx/image/Makefile

```diff
@@ -58,7 +58,7 @@ define Device/Default
   KERNEL := kernel-bin | patch-cmdline | lzma | uImage lzma
   COMPILE :=
   IMAGES := sysupgrade.bin
-  IMAGE/sysupgrade.bin = append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | check-size $$$$(IMAGE_SIZE)
+  IMAGE/sysupgrade.bin = append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
```

Comment:
This will probably break* almost all images under whole ar71xx target, please include `append-metadata` step per device/image, not here as default. We will (someday) make this default but for now, only few devices under ar71xx target uses sysupgrade images with embedded metadata.

* Variable `SUPPORTED_DEVICES` isn't reset/cleared anywhere here which might end up with sysupgrade images with wrong/unrelated metadata embedded.